src/controllers/trainee/index.ts

```diff
@@ -0,0 +1 @@
+export { default as myRouter } from './routes';
```

Comment:
export as traineeRouter for more readability.

Reply:
Done required changes.